homeassistant/components/rest_command/strings.json

```diff
@@ -0,0 +1,19 @@
+{
+  "services": {
+    "reload": {
+      "name": "[%key:common::action::reload%]",
+      "description": "Reloads RESTful commands from the YAML-configuration."
+    }
+  },
+  "exceptions": {
+    "timeout": {
+      "message": "Timeout while waiting for response from the server."
+    },
+    "client_error": {
+      "message": "An error occurred while requesting the resource."
+    },
+    "non_ok_status_code": {
+      "message": "The server answered with a {status_code} status code"
```

Comment:
```suggestion
      "message": "The server answered with a {status_code} status code."
```

Reply:
I think I should remove the dot from the other messages as most of the other integration messages don't end with a dot.